ml/src/dataset.py

```
@@ -635,6 +635,43 @@ def fold_split(fold_manifest: Mapping, repeat: int, fold: int) -> dict[str, list
     return {"train": train, "test": test}
 
 
+def withhold_from_training(
+    split: Mapping[str, list[dict]],
+    inner: Sequence[Mapping[str, list[dict]]],
```

Ruff (0.16.3)

[error] Undefined name `Sequence`

(F821)